bsc-plugin/src/lib/rooibos/MockUtil.ts

```diff
@@ -186,4 +186,41 @@ export class MockUtil {
         }
     }
 
+    public addRuntimeGlobalFunctionMocks(file: BrsFile, astEditor: Editor) {
+        file.ast.walk(createVisitor({
+            FunctionStatement: (statement, parent, owner, key) => {
+                if (!file.pkgPath.includes('rooibos/') && !file.pkgPath.endsWith('spec.brs')) {
+                    let isDisabledFoMocking = statement.annotations?.find(x => x.name.toLowerCase() === 'disablemocking');
```

Comment:
can we get a unit test that shows what the output of this function is, please? can just be like any of the tests in plugin.spect.ts; - they're more like integration tests, really; but it helps to know what we expect the final brs to look like. I have several examples of this.

Reply:
Updates tests.